.github/workflows/build.yaml

```diff
@@ -169,6 +169,8 @@ jobs:
           cmake . -B build -G Ninja \
            -DCMAKE_INSTALL_PREFIX=$PWD/build/install \
            -DCMAKE_BUILD_TYPE=Release
+          sed -i 's/LOCALEDIR/LOCALEDIRBAD/' build/utility/fc_config.h
+          echo '#define LOCALEDIR "./locale"' >> build/utility/fc_config.h
```

Comment:
This means that locally-built installers will still be broken. I'd prefer to do this in CMake, or directly in the code:
```cpp
#ifdef Q_OS_WIN
  return "./locale";
#else
  return LOCALEDIR;
#endif
```

Reply:
If I run cpack locally to create the msi nothing changes, right? Or am I missing something?